lab2/kursinis.tex

```diff
@@ -47,10 +47,29 @@ \section{Context}
 the educational experience for both students and teachers. With a user-friendly interface and a comprehensive set of features, EduPal aims to streamline the learning process, fostering a more efficient and engaging educational environment.
 
 \subsection{Functionality overview}
-List features here
 
-\subsection{Tech stack}
-List tech stack here
+At present, EduPal has these functionalities:
+
+\begin{itemize}
+    \item \textbf{Create and Manage Subjects}: Users have the ability to create new subjects and manage existing ones within the system. This includes creating topics under each subject and uploading notes in PDF format related to those topics. Additionally, users can conveniently access all their own notes as well as notes shared by peers within the system.
```

Comment:
It is verb here ("Create and Manage") but you use nouns elsewhere. Why you put info about notes here?